hypothesis-python/src/hypothesis/extra/django/_fields.py

```diff
@@ -231,6 +231,22 @@ def register_field_strategy(field_type, strategy):
     _global_field_lookup[field_type] = strategy
 
 
+def get_string_field_bounds(field):
+
+    min_size = 1
+    if getattr(field, "blank", False) or not getattr(field, "required", True):
+        min_size = 0
+    min_size = max([min_size]) + [
```

Comment:
This should probably be `max([min_size] + [...])` (close-paren too early), and the same for max_size.